lxd/storage/pool_interface.go

```diff
@@ -73,7 +73,7 @@ type Pool interface {
 	RestoreCustomVolume(volName string, snapshotName string, op *operations.Operation) error
 
 	// Custom volume migration.
-	MigrationTypes(contentType drivers.ContentType) []migration.Type
+	MigrationTypes(contentType drivers.ContentType, refresh bool) []migration.Type
```

Comment:
@monstermunchkin great commit message, makes sense. Did you rebase recently? Some changes got merged yesterday that use this function without the new argument.

Reply:
@monstermunchkin ah, just seen the updates in the future commit :)